### DISCUSSION TITLE
Tips on Tuning?

### DISCUSSION THREAD

**Discussion by BobRathke** [mKeRix/room-assistant#574] [Q&A]
Hi All!

Just got Room Assistant up & running on 3 RPI Zero's and Home Assistant. Thanks to mKeRix for your great work on this project!

As mentioned, I have 3 Pi's spread out, but one seems to be "overpowering" the other 2. They are all the same brand PI & purchased at the same time. Maybe one just has a better antenna than the others?

I am running in BT classic mode. Any suggestions for adjusting the sensitivity on these? minRSSI really does not seem to do much, they are all pretty close together with solid signals. I don't have easy access to the API to review what the actual RSSI values are.

Thanks for any tips in advance!

**Comment by mKeRix** [comment]
There isn't really a way to tune this on hardware level unfortunately, but I was thinking about adding a `factor` setting to the Bluetooth integrations. That would allow you to just manually adjust the received signal strengths on the room-assistant side to bring all of them on (close to) the same level. Would that be something you're interested in? If so: it would be cool if you could open that as a new feature request issue, just so that tracking is a bit easier for me. :)
